tools/lib/test_server.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+import time
+
+from contextlib import contextmanager
+from typing import (Any, Iterator)
+
+try:
+    import django
+    import requests
```

Comment:
I didn't split this from run-casper; this was an artifact of adding typing after the fact.